mautic/mautic
# Campaign condition based on select type field error

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Y
| New feature?  | 
| BC breaks?    | 
| Deprecations? | 
| Fixed issues  |  #1734 

## Description
When using the “equal” or “like” operator on a lead list, the condition systematically returns false.

## Steps to reproduce the bug (if applicable)

- Open the campaign builder
- Add a Lead Field condition
- Choose a select type lead field
- Set the condition to "equal"
- Observe how the condition is never fulfilled

## Steps to test this PR

Install the branch and create a new campaign with the condition above. The campaign condition should work.
